packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -136,7 +136,6 @@ const FormLabel = ({
         width,
         flexShrink: 0,
         marginRight: makeSpace(theme.spacing[5]),
-        wordBreak: 'break-all',
```

Comment:
This might be needed no? It is there to break the single word as well.

we have usecases where label might be single word `temporary_house_address` kind of label. That is breaking after the change -

Before the change
<img width="730" alt="image" src="https://user-images.githubusercontent.com/30949385/237027487-3ced9967-f181-4ffa-a05a-0281ecaa145f.png">

After the change
<img width="431" alt="image" src="https://user-images.githubusercontent.com/30949385/237027665-90c9c154-260c-4c2b-84c1-aff02ec05514.png">

Reply:
I reverted this, but I was thinking of adding a hyphen for word breaks but it doesn't  seem to work